configs/cassandra.textproto

```diff
@@ -0,0 +1,362 @@
+      "    file_checks:{"
+      "      files_to_check:{process_path:{proc_name:\"chronyd\"}}"
+      "      existence:{should_exist:true}"
+      "    }"
+      "}"
+      "check_alternatives:{"
+      "    file_checks:{"
+      "      files_to_check:{process_path:{proc_name:\"ntp\"}}"
+      "      existence:{should_exist:true}"
+      "    }"
+      "}"
+      "check_alternatives:{"
+      "    file_checks:{"
+      "      files_to_check:{process_path:{proc_name:\"systemd-timesyn\"}}"
+      "      existence:{should_exist:true}"
+      "    }"
+      "}"
+  }
+}
+
+benchmark_configs: {
+  id: "cassandra-authentication"
+  compliance_note: {
+    version: { version: "1.0.0" benchmark_document: "CIS Apache Cassandra" }
+    title: "Ensure that authentication is enabled for Cassandra databases"
+    description:
+      "Authentication is pluggable in Cassandra and is configured using the authenticator setting "
+      "in cassandra.yaml. Cassandra ships with two options included in the default distribution, "
+      "AllowAllAuthenticator and PasswordAuthenticator. The default, "
+      "AllowAllAuthenticator, performs no authentication checks and therefore requires no "
+      "credentials. It is used to disable authentication completely. The second option, "
+      "PasswordAuthenticator, stores encrypted credentials in a system table. This can be used "
+      "to enable simple username/password authentication. "
+    rationale:
+      "Authentication is a necessary condition of Cassandra’s permissions subsystem, so if "
+      "authentication is disabled then so are permissions. Failure to authenticate clients, users, "
+      "and/or servers can allow unauthorized access to the Cassandra database and can prevent "
+      "tracing actions back to their sources. The authentication mechanism should be "
+      "implemented before anyone accesses the Cassandra server. "
+    remediation:
+      "To enable the authentication mechanism: "
+      "1. Stop the Cassandra database. "
+      "2. Modify cassandra.yaml file to modify/add entry for authenticator: set it to "
+      "PasswordAuthenticator "
+      "3. Start the Cassandra database. "
+    cis_benchmark: {
+      profile_level: 1
+      severity: HIGH
+    }
+    scan_instructions:
+    "check_alternatives:{"
+    "  file_checks:{"
+    "    files_to_check:{single_file:{path:\"/etc/cassandra/cassandra.yaml\"}}"
+    "    existence:{should_exist: true}"
+    "    non_compliance_msg: \"Cassandra configuration file does not exist\""
+    "  }"
+    "  file_checks:{"
+    "    files_to_check:{single_file:{path:\"/etc/cassandra/cassandra.yaml\"}}"
+    "    content_entry: {"
+    "      match_type: NONE_MATCH"
+    "      match_criteria: {"
+    "        filter_regex: \"authenticator: .*\""
+    "        expected_regex: \"authenticator: AllowAllAuthenticator\""
+    "      }"
+    "    }"
+    "    non_compliance_msg: \"Cassandra is configured without password authentication\""
```

Comment:
I think it's fine to leave this out and use the default non-compliance message here (which would be something like `found "authenticator: AllowAllAuthenticator" in cassandra.yaml, expected not to`). That way the user will know what the scanner checked for in case there's some bug with the finding.

Same for the checks below.